doc/Grammalecte.txt

```diff
@@ -89,7 +89,11 @@ this line in you ~/.vimrc: >
 
 4.2 Installing Grammalecte
 
-To use this plugin, you need to install the Python Grammalecte program.
+To use this plugin, you need to install the Python Grammalecte program. 
+For this, unpack the OXT file into a folder of your liking, say 
+$HOME/Grammalecte-fr-v0.5.14, and point let g:grammalect_cli_py towards it:
+
+  let g:grammalect_cli_py='$HOME/Grammalecte-fr-v0.5.14/pythonpath/cli.py 
```

Comment:
There is a missing quote here. Can you correct before I merge the pull request?

I would also change "unpack the OXT file" into "unzip the OXT file", since some people may not realize that OXT is actually just a zip file.